### PR TITLE
CI Impovments

### PR DESCRIPTION
**Pull request by shurivich**
Split windows build tracer and profiler
Add retry for download and docker build tasks

@DataDog/apm-dotnet

### CODE REVIEW

File: .azure-pipelines/ultimate-pipeline.yml
Comment (andrewlock) [inline .azure-pipelines/ultimate-pipeline.yml:146]: This requires _both_ the tracer and the profiler, will need to be in a subsequent task (possibly can be done in the package windows step?)

File: .azure-pipelines/ultimate-pipeline.yml
Comment (andrewlock) [inline .azure-pipelines/ultimate-pipeline.yml:593]: 🎉

File: .azure-pipelines/ultimate-pipeline.yml
Comment (andrewlock) [inline .azure-pipelines/ultimate-pipeline.yml:127]: As we've renamed this task, we'll need to do two things before we can merge this PR:

- Update the new no-op pipeline.yml to rename `build_windows` to `build_windows_tracer`.
- Mark the `build_windows_tracer` task as required in GitHub (instead of the `build_windows` task)

**Review by andrewlock — COMMENTED** [review]
Looking good! just a couple of minor things + I think we need to move `BuildMonitoringHome` to the `package_windows` stage

File: .azure-pipelines/ultimate-pipeline.yml
Comment (shurivich) [reply .azure-pipelines/ultimate-pipeline.yml:144]: I've updated noop yml pipeline in e7c81415a45abbbac69f4297a95e88eb99739387
Still need to mark `build_windows_tracer` as required

File: .azure-pipelines/ultimate-pipeline.yml
Comment (shurivich) [reply .azure-pipelines/ultimate-pipeline.yml:163]: After conversation, resolving this comment, since `BuildMonitoringHome` is not depends on tracer and profiler.
It just a bad name and description.

File: .azure-pipelines/ultimate-pipeline.yml
Comment (andrewlock) [inline .azure-pipelines/ultimate-pipeline.yml:1721]:
```suggestion
          "$(smokeTestAppDir)"
```

File: .azure-pipelines/ultimate-pipeline.yml
Comment (andrewlock) [inline .azure-pipelines/ultimate-pipeline.yml:1759]:
```suggestion
              "$(smokeTestAppDir)"
```

**Review by andrewlock — APPROVED** [review]

**Comment by andrewlock** [comment]
## Code Coverage Report :bar_chart:

:heavy_check_mark: Merging #2306 into [master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550) will **not change** line coverage
:heavy_check_mark: Merging #2306 into [master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550) will **not change** branch coverage
:heavy_check_mark: Merging #2306 into [master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550) will **not change** complexity


|           | [master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550) | #2306       | Change   | 
|:----------|:-----------:|:-----------:|:--------:|
| Lines     | `10597` / `14831` | `10551` / `14831` |          |
| Lines %   | `71%`      | `71%`      |  `0%` :heavy_check_mark:  |
| Branches  | `5322` / `7675` | `5294` / `7675` |          |
| Branches %| `69%`      | `69%`      |  `0%` :heavy_check_mark:  |
| Complexity|   `8914`      | `8914`        |  `0`  :heavy_check_mark:    |

View the full report for further details:

* [HTML report for master](https://dev.azure.com/datadoghq/dd-trace-dotnet/_build/results?buildId=86992&view=codecoverage-tab) | [Source file](https://dev.azure.com/datadoghq/a51c4863-3eb4-4c5d-878a-58b41a049e4e/_apis/build/builds/86992/artifacts?artifactName=Code%20Coverage%20Report_86992&api-version=6.0&%24format=zip)  
* [HTML report for this PR #2306](https://dev.azure.com/datadoghq/dd-trace-dotnet/_build/results?buildId=87100&view=codecoverage-tab) | [Source file](https://dev.azure.com/datadoghq/a51c4863-3eb4-4c5d-878a-58b41a049e4e/_apis/build/builds/87100/artifacts?artifactName=Code%20Coverage%20Report_87100&api-version=6.0&%24format=zip)

### Datadog.Trace Breakdown :heavy_check_mark:

|        | [master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550) | #2306       | Change   | 
|:-------|:-----------:|:-----------:|:--------:|
| Lines %| `71%`      | `71%`       |  `0%` :heavy_check_mark:  |
| Branches %| `69%`      | `69%`       |  `0%` :heavy_check_mark:  |
| Complexity| `8914`      | `8914`       |  `0` :heavy_check_mark:  |

The following classes have significant coverage changes.

| File    | Line coverage change | Branch coverage change | Complexity change |
|:--------|:--------------------:|:----------:|:--------:|
| [Datadog.Trace.ClrProfiler.AutoInstrumentation.Couchbase.IIOServiceExecuteIntegrationBis](https://github.com/DataDog/dd-trace-dotnet/tree/4ad18a58fe484d88508dc13de600bf03bb795d5f/tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Couchbase/IIOServiceExecuteIntegrationBis.cs) | `-100%` :no_entry: | `0%` :heavy_check_mark:   | `0` :heavy_check_mark: |
| [Datadog.Trace.ClrProfiler.AutoInstrumentation.Testing.MsTestV2.TestMethodRunnerExecuteIntegration](https://github.com/DataDog/dd-trace-dotnet/tree/4ad18a58fe484d88508dc13de600bf03bb795d5f/tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Testing/MsTestV2/TestMethodRunnerExecuteIntegration.cs) | `-33%` :no_entry: | `-19%` :no_entry:   | `0` :heavy_check_mark: |
| [Datadog.Trace.Ci.Agent.CIAgentWriter](https://github.com/DataDog/dd-trace-dotnet/tree/4ad18a58fe484d88508dc13de600bf03bb795d5f/tracer/src/Datadog.Trace/Ci/Agent/CIAgentWriter.cs) | `-10%` :no_entry: | `-21%` :no_entry:   | `0` :heavy_check_mark: |
| [Datadog.Trace.ClrProfiler.AutoInstrumentation.Testing.NUnit.NUnitCompositeWorkItemSkipChildrenIntegration](https://github.com/DataDog/dd-trace-dotnet/tree/4ad18a58fe484d88508dc13de600bf03bb795d5f/tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Testing/NUnit/NUnitCompositeWorkItemSkipChildrenIntegration.cs) | `-8%` :no_entry: | `-8%` :no_entry:   | `0` :heavy_check_mark: |


View the full reports for further details:

* [HTML report for master](https://dev.azure.com/datadoghq/dd-trace-dotnet/_build/results?buildId=86992&view=codecoverage-tab) | [Source file](https://dev.azure.com/datadoghq/a51c4863-3eb4-4c5d-878a-58b41a049e4e/_apis/build/builds/86992/artifacts?artifactName=Code%20Coverage%20Report_86992&api-version=6.0&%24format=zip)  
* [HTML report for this PR #2306](https://dev.azure.com/datadoghq/dd-trace-dotnet/_build/results?buildId=87100&view=codecoverage-tab) | [Source file](https://dev.azure.com/datadoghq/a51c4863-3eb4-4c5d-878a-58b41a049e4e/_apis/build/builds/87100/artifacts?artifactName=Code%20Coverage%20Report_87100&api-version=6.0&%24format=zip)

**Review by zacharycmontoya — APPROVED** [review]

**Comment by andrewlock** [comment]
## Benchmarks Report :snail:

Benchmarks for #2306 compared to [master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550): 
* 1 benchmarks are **slower**, with geometric mean 1.156
* All benchmarks have **the same** allocations


The following thresholds were used for comparing the benchmark speeds:
* Mann–Whitney U test with statistical test for significance of **5%**
* Only results indicating a difference greater than **10%** and **0.3 ns** are considered.

Allocation changes below **0.5%** are ignored.


### Benchmark details


<details><summary><strong>Benchmarks.Trace.AgentWriterBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`WriteAndFlushEnrichedTraces`|net472|891μs|4.64μs|24.1μs|0| 0|0|3.16&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`WriteAndFlushEnrichedTraces`|netcoreapp3.1|676μs|2.41μs|9.34μs|0| 0|0|2.57&nbsp;KB|
|#2306|`WriteAndFlushEnrichedTraces`|net472|878μs|4.63μs|23.6μs|0| 0|0|3.16&nbsp;KB|
|#2306|`WriteAndFlushEnrichedTraces`|netcoreapp3.1|661μs|2.82μs|10.9μs|0| 0|0|2.57&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.AspNetCoreBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`SendRequest`|net472|0ns|0ns|0ns|0| 0|0|0&nbsp;b|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`SendRequest`|netcoreapp3.1|295μs|1.29μs|5.01μs|0.145| 0|0|19.63&nbsp;KB|
|#2306|`SendRequest`|net472|0ns|0ns|0ns|0| 0|0|0&nbsp;b|
|#2306|`SendRequest`|netcoreapp3.1|303μs|1.49μs|6.84μs|0.147| 0|0|19.63&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.DbCommandBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`ExecuteNonQuery`|net472|1.6μs|7.62ns|31.4ns|0.0923| 0.000796|0|586&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`ExecuteNonQuery`|netcoreapp3.1|1.45μs|4.43ns|16.6ns|0.00891| 0|0|624&nbsp;B|
|#2306|`ExecuteNonQuery`|net472|1.6μs|8.06ns|35.1ns|0.0926| 0.000792|0|586&nbsp;B|
|#2306|`ExecuteNonQuery`|netcoreapp3.1|1.41μs|5.93ns|23ns|0.00838| 0|0|624&nbsp;B|
</details>


<details><summary><strong>Benchmarks.Trace.ElasticsearchBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`CallElasticsearch`|net472|2.57μs|11.8ns|47.4ns|0.124| 0|0|794&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`CallElasticsearch`|netcoreapp3.1|1.58μs|5.23ns|20.2ns|0.0103| 0|0|784&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`CallElasticsearchAsync`|net472|3.1μs|12.8ns|51.3ns|0.145| 0|0|931&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`CallElasticsearchAsync`|netcoreapp3.1|1.77μs|8.64ns|38.7ns|0.0122| 0|0|904&nbsp;B|
|#2306|`CallElasticsearch`|net472|2.41μs|10.3ns|38.7ns|0.124| 0|0|794&nbsp;B|
|#2306|`CallElasticsearch`|netcoreapp3.1|1.65μs|8.52ns|44.3ns|0.011| 0|0|784&nbsp;B|
|#2306|`CallElasticsearchAsync`|net472|3.12μs|13ns|50.2ns|0.145| 0|0|931&nbsp;B|
|#2306|`CallElasticsearchAsync`|netcoreapp3.1|1.74μs|7.26ns|33.3ns|0.0124| 0|0|904&nbsp;B|
</details>


<details><summary><strong>Benchmarks.Trace.GraphQLBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`ExecuteAsync`|net472|2.83μs|12.9ns|49.9ns|0.164| 0|0|1.05&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`ExecuteAsync`|netcoreapp3.1|2.02μs|10.9ns|61.5ns|0.0144| 0|0|1.02&nbsp;KB|
|#2306|`ExecuteAsync`|net472|2.8μs|15.7ns|102ns|0.164| 0|0|1.05&nbsp;KB|
|#2306|`ExecuteAsync`|netcoreapp3.1|1.96μs|10.5ns|59.3ns|0.0145| 0|0|1.02&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.HttpClientBenchmark</strong> - Slower :warning: Same allocations :heavy_check_mark:



#### Slower :warning: in #2306

|  Benchmark         | diff/base | Base Median (ns) | Diff Median (ns) | Modality   |
|:----------|-----------:|-----------:|--------:|--------:|
| Benchmarks.Trace.HttpClientBenchmark.SendAsync&#8209;net472 | 1.156 | 6,274.51 | 7,256.14 | 

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`SendAsync`|net472|6.35μs|36.8ns|299ns|0.351| 0|0|2.26&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`SendAsync`|netcoreapp3.1|4.43μs|17.7ns|68.6ns|0.0285| 0|0|2.16&nbsp;KB|
|#2306|`SendAsync`|net472|7.22μs|34.9ns|140ns|0.35| 0|0|2.26&nbsp;KB|
|#2306|`SendAsync`|netcoreapp3.1|4.51μs|23.4ns|110ns|0.0301| 0|0|2.16&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.ILoggerBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`EnrichedLog`|net472|3.69μs|15.9ns|61.7ns|0.225| 0|0|1.44&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`EnrichedLog`|netcoreapp3.1|3.3μs|16.1ns|68.4ns|0.0211| 0|0|1.52&nbsp;KB|
|#2306|`EnrichedLog`|net472|3.34μs|16.6ns|76ns|0.224| 0|0|1.44&nbsp;KB|
|#2306|`EnrichedLog`|netcoreapp3.1|3.11μs|11.1ns|42.9ns|0.0216| 0|0|1.52&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.Log4netBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`EnrichedLog`|net472|291μs|1.31μs|5.06μs|0.432| 0.144|0|4.31&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`EnrichedLog`|netcoreapp3.1|241μs|914ns|3.54μs|0| 0|0|4.19&nbsp;KB|
|#2306|`EnrichedLog`|net472|292μs|877ns|3.28μs|0.441| 0.147|0|4.31&nbsp;KB|
|#2306|`EnrichedLog`|netcoreapp3.1|240μs|820ns|3.07μs|0| 0|0|4.19&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.NLogBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`EnrichedLog`|net472|7.86μs|38.9ns|165ns|0.502| 0|0|3.21&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`EnrichedLog`|netcoreapp3.1|6.43μs|23.3ns|90.2ns|0.0479| 0|0|3.58&nbsp;KB|
|#2306|`EnrichedLog`|net472|7.39μs|18.7ns|67.3ns|0.502| 0|0|3.21&nbsp;KB|
|#2306|`EnrichedLog`|netcoreapp3.1|6.62μs|35.5ns|191ns|0.0514| 0|0|3.58&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.RedisBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`SendReceive`|net472|2.14μs|8.37ns|31.3ns|0.158| 0|0|1&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`SendReceive`|netcoreapp3.1|1.97μs|7.33ns|28.4ns|0.0137| 0|0|1&nbsp;KB|
|#2306|`SendReceive`|net472|2.25μs|12.2ns|70ns|0.157| 0|0|1&nbsp;KB|
|#2306|`SendReceive`|netcoreapp3.1|1.98μs|11.1ns|70ns|0.014| 0|0|1&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.SerilogBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`EnrichedLog`|net472|6.04μs|28ns|108ns|0.288| 0|0|1.85&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`EnrichedLog`|netcoreapp3.1|5.31μs|25.8ns|110ns|0.0209| 0|0|1.47&nbsp;KB|
|#2306|`EnrichedLog`|net472|6.25μs|33.1ns|172ns|0.288| 0|0|1.85&nbsp;KB|
|#2306|`EnrichedLog`|netcoreapp3.1|5.46μs|21.7ns|81.1ns|0.0196| 0|0|1.47&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.SpanBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`StartFinishSpan`|net472|949ns|5.29ns|32.6ns|0.0703| 0|0|449&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`StartFinishSpan`|netcoreapp3.1|918ns|4.5ns|18ns|0.00599| 0|0|448&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`StartFinishScope`|net472|1.16μs|6.43ns|38ns|0.0832| 0|0|530&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/3fc3c14ff2276ad68f3290b7d2dd6707e6420550)|`StartFinishScope`|netcoreapp3.1|1.05μs|3ns|11.6ns|0.00787| 0|0|568&nbsp;B|
|#2306|`StartFinishSpan`|net472|939ns|4.26ns|15.4ns|0.0704| 0|0|449&nbsp;B|
|#2306|`StartFinishSpan`|netcoreapp3.1|946ns|3.9ns|15.1ns|0.00605| 0|0|448&nbsp;B|
|#2306|`StartFinishScope`|net472|1.13μs|5.02ns|18.1ns|0.0831| 0|0|530&nbsp;B|
|#2306|`StartFinishScope`|netcoreapp3.1|1.05μs|3.42ns|13.2ns|0.00757| 0|0|568&nbsp;B|
</details>

File: .azure-pipelines/ultimate-pipeline.yml
Comment (lucaspimentel) [reply .azure-pipelines/ultimate-pipeline.yml:144]: I made `build_windows` no longer required so it doesn't block this PR.

File: .azure-pipelines/ultimate-pipeline.yml
Comment (lucaspimentel) [reply .azure-pipelines/ultimate-pipeline.yml:163]: > It just a bad name and description.

Should we fix that? 😅

**Review by lucaspimentel — APPROVED** [review]
Thanks!

**Pull request merged**
